src/Bridge/Symfony/DependencyInjection/SonataExporterExtension.php

```diff
@@ -39,7 +39,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
     private function configureExporter(ContainerBuilder $container, array $config): void
     {
-        foreach (['csv', 'json', 'xls', 'xml'] as $format) {
+        foreach (['csv', 'json', 'xls', 'xlsx', 'xml'] as $format) {
```

Comment:
Not related to this PR, but IMO we should add the tags in the definitions instead of updating two places every time we add a writer, I'll make a PR.